### ISSUE TITLE
RSS Feed for filb.de seems to bug out ownCloud News

### ISSUE THREAD

**Issue by niemalsnever** [owncloud/news#878]
Hi,
for some reason the feed for filb.de (http://feeds.feedburner.com/filb/) bugs out ownCloud news. Some elements keep reoccuring forever and ever, and new elements don't show up in the feed at all. When first added, the feed seems to work fine but after the first refresh cycle (through cron) the problem with the reoccurring elements shows up. I can't say if anything is wrong with the feed, but it seems to work in other feed readers.

niemalsnever

**Comment by BernhardPosselt** [comment]
First of all, the feed does not provide a guid elmenet so we can not detect if it is the same entry. The next thing that is checked is if the body, title and url are the same. 

Try to reload the feed url in chrome using CTRL + F5 a few times and you will see that the content changes after a few times. 

TL;DR: The feed is changing its item content and provides no guid which basically makes it impossible to properly detect if its the same item. I can only imagine that other feed readers use some sort of exception for this.

Please contact the guy behind the feed and ask him to add a guid element (he can reuse the link)
